controller:
  update_frequency: 10     # [hz]
  k_p: 0.030                 # []
  k_d: 0.00001                 # []
  max_vel_cmd: 2.5            # [rad/s]

  test_yaw_rate: 2.0        # [rad/s]

line:
  theta_upper: deg(8)      # [rad]
  theta_lower: deg(-8)     # [rad]
  line_vel_threshold: 40    # [px/s]
  line_min_length: 60       # [px]